packages/database/src/schema.prisma

```diff
@@ -147,27 +152,25 @@ model DocumentCategoryStatus {
 }
 
 model ContactDetails {
-  id               String        @id @default(dbgenerated("newid()")) @db.UniqueIdentifier
-  firstName        String
-  lastName         String
-  emailAddress     String
-  phone            String
-  fax              String?
-  organisation     String
-  addressId        String        @db.UniqueIdentifier
-  Address          FullAddress   @relation(fields: [addressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
-  paymentMethodId  String
-  PaymentMethod    PaymentMethod @relation(fields: [paymentMethodId], references: [id], onDelete: NoAction, onUpdate: NoAction)
```

Comment:
moving this isn't going to break existing data is it? Are we going to need to clear dev and test data again?